src/System.Memory/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Date.G.cs

```diff
@@ -2,8 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Diagnostics;
```

Comment:
Personally, I don't think we need go out of our way to remove `using System.Diagnostics`. Encouraging the use of assertions by removing friction from its use is more valuable than removing every bit of white-font from the IDE.